spec/lib/miq_automation_engine/service_methods/miq_ae_service_miq_user_role_spec.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeServiceMiqUserRoleSpec
+  describe MiqAeMethodService::MiqAeServiceMiqUserRole do
+    before(:each) do
+      @role = FactoryGirl.create(:miq_user_role, :name => "Role1")
+    end
+
+    it "check name" do 
+      expect(@role.name).to eq("Role1")
```

Comment:
@jockey10 
This is checking the name of the miq_user_role object not the role for the service model object 
MiqAeMethodService::MiqAeServiceMiqUserRole

You could replace the whole spec to look something like this
```
module MiqAeServiceMiqUserRoleSpec
  describe MiqAeMethodService::MiqAeServiceMiqUserRole do
    let(:role) { FactoryGirl.create(:miq_user_role, :name => 'Role1') }
    let(:svc_role) { MiqAeMethodService::MiqAeServiceMiqUserRole.find(role.id) }

    it "#name" do
       expect(svc_role.name).to eq(role.name)
    end
end
```